src/App.js

```diff
@@ -1,30 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { Switch, Route } from 'react-router-dom';
 import FirebaseTesting from './pages/FirebaseTest';
```

Comment:
It looks like in the merge that `Redirect` got removed from the imports from `react-router-dom`. That might be causing the build error.